src/mlpack/bindings/R/CMakeLists.txt

```diff
@@ -314,6 +309,14 @@ macro (post_r_setup)
   file(GLOB_RECURSE R_SRC_CPP_FILES RELATIVE "${CMAKE_CURRENT_SOURCE_DIR}" *.cpp)
   set(R_SRC_FILES ${R_SRC_HPP_FILES} ${R_SRC_CPP_FILES})
 
+  # In Case of R_Bindings Move all of these header to <package>/src/cereal/.
+  # Collect all header files in the library.
```

Comment:
It might be worth pointing out that we do this because the currently-packaged version of Rcereal throws warnings on CRAN?

Reply:
Done.